gitopsaddon/test-cycle-eks-ocp.sh

```
@@ -0,0 +1,457 @@
+    for cluster in "${CLUSTER_NAMES[@]}"; do
+        # Hub: MCA gone
+        if hub get managedclusteraddon -n "$cluster" gitops-addon &>/dev/null; then
+            log "CLEANUP ISSUE: MCA still in $cluster"; issues=$((issues+1))
+        else
+            pass "Hub: MCA deleted for $cluster"
+        fi
+
+        # Managed cluster: ArgoCD CR removed (wait up to 60s)
+        local aw=0
+        while [ "$aw" -lt 60 ]; do
+            on_cluster "$cluster" get argocd acm-openshift-gitops -n openshift-gitops &>/dev/null 2>/dev/null || break
```

Shellcheck (0.11.0)

[error] Multiple redirections compete for stderr. Use cat, tee, or pass filenames instead.

(SC2261)